src/toil/batchSystems/kubernetes.py

```diff
@@ -460,25 +467,10 @@ def issueBatchJob(self, jobNode):
             logger.debug('Launched job: %s', jobName)
             
             return jobID
-            
-            
-    def _isJobOurs(self, jobObject):
-        """
-        Determine if a Kubernetes job belongs to us.
-        
-        :param kubernetes.client.V1Job jobObject: a Kubernetes job being considered.
-
-        :return: True if the job is our responsibility, and false otherwise.
-        :rtype: bool
-        """
-        
-        return jobObject.metadata.name.startswith(self.jobPrefix)
-        
-        
     
-    def _ourJobObjects(self, onlySucceeded=False, limit=None):
+    def _ourJobObjectList(self, onlySucceeded=False, limit=None):
```

Comment:
It looks like we no longer actually implement a limit, so you should remove the limit option here.